### PR TITLE
8303413: (fs) Ignore polling interval sensitivity modifiers in PollingWatchService

### PR DESCRIPTION
**Pull request by bplb**
Modify `sun.nio.fs.PollingWatchService` to ignore any polling sensitivity modifiers as done by the other `WatchService`s.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Change requires CSR request [JDK-8303471](https://bugs.openjdk.org/browse/JDK-8303471) to be approved
- [x] Commit message must refer to an issue

### Issues
 * [JDK-8303413](https://bugs.openjdk.org/browse/JDK-8303413): (fs) Ignore polling interval sensitivity modifiers in PollingWatchService
 * [JDK-8303471](https://bugs.openjdk.org/browse/JDK-8303471): (fs) Ignore polling interval sensitivity modifiers in PollingWatchService (**CSR**)

### Reviewers
 * [Alan Bateman](https://openjdk.org/census#alanb) (@AlanBateman - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12795/head:pull/12795` \
`$ git checkout pull/12795`

Update a local copy of the PR: \
`$ git checkout pull/12795` \
`$ git pull https://git.openjdk.org/jdk pull/12795/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12795`

View PR using the GUI difftool: \
`$ git pr show -t 12795`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12795.diff">https://git.openjdk.org/jdk/pull/12795.diff</a>

</details>

### CODE REVIEW

**Comment by AlanBateman** [comment]
Test change look okay but it will need a CSR and RN to track the behavior change. Also test/jdk/java/nio/file/WatchService/SensitivityModifier.java will need attention - maybe it should just be replaced with a sanity test that ensures that the register method does not throw when called with one of these modifiers.

**Comment by bplb** [comment]
> Also test/jdk/java/nio/file/WatchService/SensitivityModifier.java will need attention

I ran this test already and it still succeeds.

**Comment by AlanBateman** [comment]
> I ran this test already and it still succeeds.

I'm sure it does, but shouldn't it be replaced with some simple to just make sure that the modifiers are ignored? It is currently one of the slowest tests in the jdk_nio tree and with the changes here it overlaps with the existing WatchService tests.

File: test/jdk/java/nio/file/WatchService/SensitivityModifier.java [hunk 1]
```diff
@@ -93,51 +91,6 @@ static void doTest(Path top) throws Exception {
 
             // register the directories (random sensitivity)
             register(dirs, watcher);
-
```
Comment (AlanBateman) [inline test/jdk/java/nio/file/WatchService/SensitivityModifier.java:96]: I assume there is a lot more that is no longer needed in this test, e.g most of doTest is not needed, POLL_TIMEOUT_SECONDS can be removed, ...  I suspect the resulting test will be very simple - just test that the temp dir can be registered with each of the sensitivity levels.

Comment (bplb) [reply test/jdk/java/nio/file/WatchService/SensitivityModifier.java:96]: Fixed (enough?) in 2e9867bd45cfbad727699a1254514227b995dd7a.

**Review by AlanBateman — APPROVED** [review]
Updated version looks fine. In this case, I'd probably update the @bug line to add 8303413, only because the test is very different to the original test.

**Comment by bplb** [comment]
> I'd probably update the @bug line to add 8303413

Done in a2216a728b642a649c4c6261cd8ed1b1bc18136a.

**Review by AlanBateman — APPROVED** [review]

**Pull request closed without merging**